src/IdentityServer/appsettings.conf

```diff
@@ -201,3 +201,8 @@
 
 # Logging__LogLevel__Microsoft.Hosting.Lifetime = Information
 # Logging__LogLevel__Microsoft = Warning
+
+####################
+## Application Insights
+####################
+#ConnectionString =
```

Comment:
What do you think about making this it's own config section? `Insights__ConnectionString = `; that keeps the setting isolated and allows for adding other Insight options without having to recompile anything.

Then you could pass that config section...
```cs
if (string.IsNullOrEmpty(Configuration["Insights:ConnectionString"]).Equals(false))
                services.AddApplicationInsightsTelemetry(Configuration.GetSection("Insights"));
```